www/js/draw-model.js

```diff
@@ -189,6 +197,12 @@ define([
 					this.addNodeDescription(node.ID);
 				}
 			}
+			if(node.variableType === "unknown" && cachedNode.variableType !== "unknown"){
+				this._unknownQuantityNodeCount++;
```

Comment:
Sorry this code shouldn't be here. The node counts should be a property of the model, so all counting related code should be there. The logic for whatever the count is. And then it should be shown in draw-model.